packages/webapp/src/pages/IntegrationCreate.tsx

```diff
@@ -333,14 +334,13 @@ export default function IntegrationCreate() {
                                             </Tooltip>
                                         </div>
                                         <div className="mt-1">
-                                            <input
+                                            <TagsInput
```

Comment:
Could you please edit the tooltip help a couple lines above (327/328) from 

"Comma-separated ('scope1,scope2,scope3').\n Scopes are documented on the developer portal of the Integration Provider."

to 

"The list of scope should be found in the documentation of the external provider."